### PR TITLE
Change the package name from alchemist to alchemist_py

### PR DESCRIPTION
**Pull request by ohkawatks**
The old package name "alchemist" is causing a problem.
In the installation instruction, the "checkinstall" does not work in my environment (Ubuntu18.04). So it is changed to more general "make install".

### CODE REVIEW

File: doc/en/Common.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+- g++, make
+- CMake
+- Git
+- JDK8
+- Gradle
+
+```
+$ sudo apt install build-essential cmake git openjdk-8-jdk gradle -y
+$ sudo apt install libncurses5 -y
```
Comment (Kenta11) [inline doc/en/Common.md:28]: Thank you for your request!

Please let me confirm about libncurses5.
Is this required to run alchemist, Vivado or VivadoHLS?
I'm not sure if libncurses5 is required while using alchemist.

File: doc/en/Common.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+```
+
+## Vivado, Vivado HLS
+
+Refer to [Vivado Design Suite User Guide UG973(v2019.1)](https://www.xilinx.com/support/documentation/sw_manuals/xilinx2019_1/ug973-vivado-release-notes-install-license.pdf).
+
+## alchemist_py
+
+```
+$ git clone https://github.com/ohkawatks/alchemist_py
```
Comment (Kenta11) [inline doc/en/Common.md:52]: The URL is your repository.
Do you mind if I fix it later?

**Comment by Kenta11** [comment]
Everything else is perfect. I am going to merge commits after I get a reply from you.

File: doc/en/Common.md (hunk 2 above)
Comment (ohkawatks) [reply doc/en/Common.md:52]: No, I don't. Please fix it to the correct one.

File: doc/en/Common.md (hunk 1 above)
Comment (Kenta11) [reply doc/en/Common.md:28]: [libncurses5 is required by Vivado for simulation.](https://www.xilinx.com/support/answers/63794.html) However, Fast-RTPS does not require it. 

Therefore, I will remove the line.

Comment (ohkawatks) [reply doc/en/Common.md:28]: Thanks!

File: doc/en/Common.md (hunk 2 above)
Comment (ohkawatks) [reply doc/en/Common.md:52]: F.Y.I.) I made a clean Ubuntu18.04 environment and confirmed the following commands works:

```
ohkawa@alchemist:~$ sudo apt install python3-pip
ohkawa@alchemist:~$ sudo pip3 install git+https://github.com/Kenta11/alchemist_py
```

**Pull request merged**